README.md

```diff
@@ -20,15 +20,22 @@ Through integration with [dask](https://dask.org/), GCM-Filters enables parallel
 
 ### Installation
 
-GCM-Filters can be installed with pip:
+GCM-Filters can be installed using conda:
+```shell
+conda install -c conda-forge gcm_filters
+```
+If you want to run the example notebooks in the [GCM-Filters documentation](https://gcm-filters.readthedocs.io/), you will need a few extra dependencies that you can install via:
+```shell
+conda install -c conda-forge gcm_filters[examples]
```

Comment:
I have to admit, I am not even sure if conda allows this syntax? I have only ever seen these 'variants' with pip.